app-shells/mksh/mksh-54.recipe

```diff
@@ -36,7 +36,6 @@ REQUIRES="
 BUILD_REQUIRES="
 	haiku_devel
 	"
-
```

Comment:
Yes, after removing my faulty patch, there really is nothing else to change, is there :) I'll find another recipe